testutil/exec.go

```diff
@@ -48,7 +48,11 @@ func MockCommand(c *check.C, basename, script string) *MockCmd {
 	logFile := path.Join(binDir, basename+".log")
 	err := ioutil.WriteFile(exeFile, []byte(fmt.Sprintf(""+
 		"#!/bin/sh\n"+
-		"echo \"$@\" >> %q\n"+
+		"printf \"$(basename \"$0\")\\n\" >> %[1]q\n"+
+		"for arg in \"$@\"; do\n"+
+		"    printf -- \"$arg\\n\" >> %[1]q\n"+
```

Comment:
zyga:~/workspace (maint-simplification) $ printf -- "I wanted to print percent-es >%s< (it should be there) \n"
I wanted to print percent-es >< (it should be there)